teacher.rb

```diff
@@ -26,15 +18,6 @@ def teach_stuff
     response
   end
 
-  def salary=(new_salary)
-    puts "This better be good!"
-    @salary = new_salary
-  end
-
-  def receive_raise(raise)
-    @salary += raise
-  end
-
   def set_performance_rating(rating)
```

Comment:
The `ApprenticeTeacher` has a very similar method with only one difference.  How could you DRY up that repeated code between the two?